### PR TITLE
RUMM-3242 add Roku integration tile

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

Adds Datadog RUM integration tile for Roku apps.

### Motivation

Surfacing the existing Roku RUM integration

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: rum_roku/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Android Integration
```
Comment (fuzzybinary) [inline rum_roku/README.md:1]:
```suggestion
# Roku Integration
```

File: rum_roku/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The Android integration does not include any metrics. To generate custom metrics from your RUM application, see [Generate Metrics][5]. 
+
+### Events
+
+For more information about events and attributes, see [RUM Roku Data Collected][6].
+
+### Service Checks
+
+The AnRokudroid integration does not include any service checks.
```
Comment (fuzzybinary) [inline rum_roku/README.md:44]:
```suggestion
The Roku integration does not include any service checks.
```

**Review by fuzzybinary — CHANGES_REQUESTED** [review]

**Review by alai97 — APPROVED** [review]
Copy looks fantastic, thanks!

File: rum_roku/manifest.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor Roku channels and generate metrics using Datadog RUM",
+    "title": "Roku",
+    "media": [],
+    "classifier_tags": [
+      "Category::Metrics",
+      "Category::TV",
+      "Category::Tracing",
+      "Supported OS::Roku"
```
Comment (alai97) [inline rum_roku/manifest.json:18]:
```suggestion
```

File: rum_roku/manifest.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor Roku channels and generate metrics using Datadog RUM",
+    "title": "Roku",
+    "media": [],
+    "classifier_tags": [
+      "Category::Metrics",
+      "Category::TV",
+      "Category::Tracing",
```
Comment (alai97) [inline rum_roku/manifest.json:17]:
```suggestion
      "Category::Tracing"
```

**Review by ofek — APPROVED** [review]

**Review by fuzzybinary — APPROVED** [review]

**Pull request merged**